client/app/bundles/course/assessment/submission/containers/TestCaseView.jsx

```diff
@@ -256,8 +257,8 @@ class VisibleTestCaseView extends Component {
           testCases.evaluation_test,
           VisibleTestCaseView.renderTitle('evaluationTestCases', graderView)
         )}
-        {graderView && !collapsible ? VisibleTestCaseView.renderOutputStream('standardOutput', testCases.stdout) : null}
-        {graderView && !collapsible ? VisibleTestCaseView.renderOutputStream('standardError', testCases.stderr) : null}
+        {(showOutputStreams && !collapsible) ? VisibleTestCaseView.renderOutputStream('standardOutput', testCases.stdout, !showStdoutAndStderr) : null}
```

Comment:
Perhaps you can also do the same patten (similar to the one in the above comment), by defining a boolean to render the output stream.